azure-pipelines.yml

```diff
@@ -6,11 +6,11 @@ jobs:
     matrix:
       Ubuntu1804:
         os: 'ubuntu:18.04'
-        command: './lint.sh'
+        command: './test.sh'
```

Comment:
Did you intend to remove lint.sh

Reply:
Yes, I'm calling it from `test.sh`

Reply:
We usually don't do that. zivid-python, zivid-ros and gcps runs multiple entry scripts in the CI file. Not a big difference, but then you can run individual scripts on-demand.